doc/VIGNETTE-GWAS-TUTORIAL.rst

```diff
@@ -0,0 +1,227 @@
+		gt=batch${batch}_chr${chrom} \
+		ref=ensembletr_refpanel_v3_chr${chrom}.bref3 \
+		ap=true \
+		out=batch${batch}_chr${chrom}_imputed \
+		map=beagle_chr${chrom}_b38.map
+	tabix -p vcf batch${batch}_chr${chrom}_imputed.vcf.gz
+
+
+Some important note about the above Beagle command:
+
+* We have had success running Beagle on batches of 1000 samples using 25GB of memory. For larger batch sizes you may need to increase the memory.
+* For the reference panel (:code:`ref` argument): you can provide either a VCF or BREF3 file. The VCF/BREF3 files for the EnsembleTR panel have equivalent information. In accordance with the Beagle recommendations, we have found the BREF3 files result in improved run times (about 10%).
+* The :code:`ap` argument is optional, but is important to include if you want to take imputation uncertainty into account when performing GWAS. This argument tells Beagle to output allele probabilities for each imputed call. Including it does increase the size of the output VCF files.
+
+To run imputation on the full cohort, you will need to run the above command on each of the VCF files generated in step 1.1 (one file per chrom per batch of samples).
+
+.. _step1_3:
+
+
+Step 1.3: Extracting TRs from each batch
+________________________________________
+
+The Beagle output files contain both the original SNP/indel genotypes (which will now be phased) in addition to the phased imputed TR genotypes. In many cases for downstream steps we are just interested in the TR genotypes (e.g. if you already ran GWAS on the SNPs/indels separately). Before the next step you can optionally filter the Beagle VCF files to include just the TRs, which will also make the files way smaller and easier to work with::
+
+	bcftools view -i 'ID~"EnsTR"' batch${batch}_chr${chrom}_imputed.vcf.gz \
+		-Oz -o batch${batch}_chr${chrom}_imputed_TRs.vcf.gz
+    	tabix -p vcf batch${batch}_chr${chrom}_imputed_TRs.vcf.gz
+
+
+This command assumes you are using the EnsembleTR reference panel, which has TR IDs of the form :code:`EnsTR:CHROM:POS`. If you are using a reference panel with different IDs for the TRs you'll need to modify the command above.
+
+On the other hand, if you want to process SNPs+TRs together in the GWAS you can skip this step.
+
+
+.. _step1_4:
+
+
+Step 1.4: Merging results from each batch
+_________________________________________
+
+
+Before downstream steps we would like to merge the VCFs across all batches to have a single VCF file per chromosome. While TRTools does include the :code:`mergeSTR` tool which can be used for this, we recommend using :code:`bcftools merge` for large cohorts when using imputed TRs. This is because: (1) after imputation with Beagle the set of REF/ALT alleles should be identical across each batch and match the reference panel, so there is not a need to use mergeSTR which specifically deals with managing differences in alleles represented across files and (2) bcftools is much faster. To merge batches::
```

Comment:
Here is a suggested change on top of Arya's:
```suggestion
Before downstream steps we would like to merge the VCFs across all batches to have a single VCF file per chromosome. While TRTools does include the :code:`mergeSTR` tool which can be used for this, we recommend using :code:`bcftools merge` for large cohorts when using imputed TRs. This is because: (1) bcftools is much faster, and (2) the set of REF/ALT alleles from Beagle should be identical across each batch and match the reference panel. (mergeSTR can handle when alleles have been represented differently across files while bcftools cannot, but that is not the case here). To merge batches::
```